cves/kernel/CVE-2017-0861.yml

```diff
@@ -280,8 +287,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: True
+  note: "This memory damaging vulnerability which can cause the entire system to crash."
```

Comment:
Can you clarify this point? How does the memory-damaging vulnerability specifically relate to inter-process communication?